jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java

```diff
@@ -419,6 +423,7 @@ public void beforeExamples(List<String> steps, ExamplesTable table) {
         }
         print(format("examplesStepsEnd", EMPTY));
         print(formatTable(table));
+        print(format("examplesStart", EMPTY));
```

Comment:
if we have `examplesStart` then we should also have `examplesEnd`, shouldn't we?

Reply:
renamed to examplesHeader to not confuse